.github/workflows/test_comprehensive.yml

```diff
@@ -171,7 +171,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: "3.x"
```

Comment:
As we run here `tox -e py39-linux-pyside2-examples`, then python 3.9 need to be installed

Reply:
Sure.

```suggestion
          python-version: 3.9
```